### PR TITLE
add todo list

### PR DESCRIPTION
**Pull request by wangxingjun778**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ FaceChain is powered by [ModelScope](https://github.com/modelscope/modelscope).
 # News
 - Colab notebook is available now! You can experience FaceChain directly with our [Colab Notebook](https://colab.research.google.com/drive/1cUhnVXseqD2EJiotZk3k7GsfQK9_yJu_?usp=sharing).
 
+# To-Do List
+- Plug and play with existing style models (e.g. style models on the Civitai)    --50%
```
Comment (sunbaigui) [inline README.md:26]: plug in off the shelf style models (e.g. style models on the Civitai)

File: README.md [hunk 2]
```diff
@@ -22,6 +22,16 @@ FaceChain is powered by [ModelScope](https://github.com/modelscope/modelscope).
 # News
 - Colab notebook is available now! You can experience FaceChain directly with our [Colab Notebook](https://colab.research.google.com/drive/1cUhnVXseqD2EJiotZk3k7GsfQK9_yJu_?usp=sharing).
 
+# To-Do List
```
Comment (yingdachen) [inline README.md:25]: Support existing style models (such as those on Civitai) in a plug-an-play fashion.  
Support customizable prompts (try on different outfits etc.)
Support customizable poses, with controlnet or composer
Support more beauty-retouch effects
Support latest foundatation models such as SDXL
Provide Colab compatability
Provide WebUI compatability

另外50%都改成开发中，100%改成已完成。

File: README_ZH.md [hunk 3]
```diff
@@ -24,6 +24,17 @@ FaceChain的模型由[ModelScope](https://github.com/modelscope/modelscope)开
...
 - Colab notebook安装已支持，您可以直接打开链接体验FaceChain: [Colab Notebook](https://colab.research.google.com/drive/1cUhnVXseqD2EJiotZk3k7GsfQK9_yJu_?usp=sharing)
 
+
+# 待办事项
+- 现成风格模型即插即用（以C站风格模型为例）   --50%
+- 增加个性化prompt模块（试穿不同的衣服等）   --50%
+- controlnet指定pose
+- 增加个性化美颜特效模块
+- 基模升级，SD 1.5升级为SD XL
+- 兼容colab使用                         --100%
```
Comment (yingdachen) [inline README_ZH.md:34]: Colab

File: README_ZH.md [hunk 4]
```diff
@@ -24,6 +24,17 @@ FaceChain的模型由[ModelScope](https://github.com/modelscope/modelscope)开
 # News
 - Colab notebook安装已支持，您可以直接打开链接体验FaceChain: [Colab Notebook](https://colab.research.google.com/drive/1cUhnVXseqD2EJiotZk3k7GsfQK9_yJu_?usp=sharing)
 
+
+# 待办事项
+- 现成风格模型即插即用（以C站风格模型为例）   --50%
+- 增加个性化prompt模块（试穿不同的衣服等）   --50%
+- controlnet指定pose
+- 增加个性化美颜特效模块
+- 基模升级，SD 1.5升级为SD XL
```
Comment (yingdachen) [inline README_ZH.md:33]: SDXL

File: README_ZH.md [hunk 5]
```diff
@@ -24,6 +24,17 @@ FaceChain的模型由[ModelScope](https://github.com/modelscope/modelscope)开
...
+- 兼容colab使用                         --100%
+- 兼容webui使用
```
Comment (yingdachen) [inline README_ZH.md:35]: WebUI

File: README_ZH.md [hunk 6]
```diff
@@ -24,6 +24,17 @@ FaceChain的模型由[ModelScope](https://github.com/modelscope/modelscope)开
 # News
 - Colab notebook安装已支持，您可以直接打开链接体验FaceChain: [Colab Notebook](https://colab.research.google.com/drive/1cUhnVXseqD2EJiotZk3k7GsfQK9_yJu_?usp=sharing)
 
+
+# 待办事项
+- 现成风格模型即插即用（以C站风格模型为例）   --50%
+- 增加个性化prompt模块（试穿不同的衣服等）   --50%
+- controlnet指定pose
```
Comment (yingdachen) [inline README_ZH.md:31]: 基于controlnet或composer等方式，支持指定poses

File: README.md (hunk 2 above)
Comment (sunbaigui) [reply README.md:25]: ok

**Review by yingdachen — APPROVED** [review]

**Pull request merged**